src/Passes/examples/TeleportChain/Makefile

```diff
@@ -15,3 +15,5 @@ QSharpVersion/qir/Example.ll:
 clean:
 	cd QSharpVersion && make clean
 
+
+# --no-disable-reference-counting --no-disable-reference-counting --no-disable-alias-counting --no-disable-string-support --no-optimise-branch-quatum-one --no-optimise-branch-quatum-zero --no-use-static-qubit-array-allocation --no-use-static-qubit-allocation --no-use-static-result-allocation
```

Comment:
What are these? This comment needs a description of what these are or should be removed.